rllib/evaluation/metrics.py

```diff
@@ -147,8 +147,8 @@ def summarize_episodes(
     if new_episodes is None:
         new_episodes = episodes
 
-    episodes, estimates = _partition(episodes)
-    new_episodes, _ = _partition(new_episodes)
+    episodes, _ = _partition(episodes)
```

Comment:
Was this a bug? Can we add a one-line comment on why we are ignoring the estimates on `episodes`, but not `new_episodes`.